hive_agent/tools/agent_db/nl_query.py

```diff
@@ -31,7 +28,7 @@ def text_2_sql(db_url: str, prompt: str, tables=None):
     sql_database = SQLDatabase(engine, include_tables=tables)
 
     query_engine = NLSQLTableQueryEngine(
-        sql_database=sql_database, tables=["entries"], llm=llm
+        sql_database=sql_database, tables=["entries"], llm=Settings.llm
```

Comment:
good catch but this should come from the config. or does this handle that automatically?

Reply:
Isn't settings global?